tests/test_utils/asserts.py

```diff
@@ -71,3 +72,33 @@ def assert_queries_count(expected_count, message_fmt=None):
                                  "The current number is {current_count}."
     message = message_fmt.format(current_count=result.count, expected_count=expected_count)
     assert expected_count == result.count, message
+
+
+class CatchJinjaParmasResult:
+    def __init__(self):
+        self.template_name = None
+        self.template_params = None
+
+
+@contextmanager
+def catch_jinja_params():
```

Comment:
And rather than a separate method/in utils, I would suggest making this a method on the already existing  TestBase class -- after all it is only useful if you have a flask app, and TestBase sets that up for us.